overlord/snapstate/flags.go

```diff
@@ -60,6 +60,10 @@ type Flags struct {
 
 	// IsAutoRefresh is true if the snap is currently auto-refreshed
 	IsAutoRefresh bool `json:"is-auto-refresh,omitempty"`
+
+	// NoReRefresh forces a refresh to skip adding "rerefresh" tasks.
+	// This is important for the Remodel() use-case.
```

Comment:
Rather than saying this is important for a particular piece of code it would be useful to say why this flag is useful:
```suggestion
	// This allows refresh to work offline, as long as refresh assets are cached.
```